docs/rules/mustache-curly-spacing.md

```diff
@@ -0,0 +1,67 @@
+# Enforce spacing on the style of mustache curly brackets. (mustache-curly-spacing)
```

Comment:
Regarding rule name I have another proposition: `mustache-interpolation-spacing`. We all know what interpolation is, plus mustache interpolation type is well described in [official documentation](https://vuejs.org/v2/guide/syntax.html#Text) - this way we should avoid any confusion. What do you think @armano2 @mysticatea ?